### ISSUE TITLE
ReferenceError: Property 'BigInt' doesn't exist, js engine: hermes 

### ISSUE THREAD

**Issue by jackz123** [helium/maker-starter-app#62]
Hi Teams
When I upgraded @helium/react-native-sdk version from 1.0.0 to 2.0.14, I got this error (ReferenceError: Property 'BigInt' doesn't exist, js engine: hermes ). Is there a solution now?

Error: Requiring module "node_modules/@helium/react-native-sdk/src/index.tsx", which threw an exception: ReferenceError: Property 'BigInt' doesn't exist, js engine: hermes 

![image](https://user-images.githubusercontent.com/16610288/225680078-39ceed9a-9e64-4fd8-84c7-c10140e73717.png)
![image](https://user-images.githubusercontent.com/16610288/225680626-fce6741a-73c1-44b7-b069-672b23ca4ce5.png)
![image](https://user-images.githubusercontent.com/16610288/225680692-5ba7c992-17fa-4c78-a84f-4a2475347faa.png)

**Comment by zhaichenyang001** [comment]
Project can't be compiled,Can you fix that,The issue as follows:
warning " > @helium/react-native-sdk@2.0.14" has incorrect peer dependency "@helium/crypto-react-native@^3.9.0".
warning " > @helium/react-native-sdk@2.0.14" has incorrect peer dependency "@helium/currency@^3.25.1".
warning " > @helium/react-native-sdk@2.0.14" has incorrect peer dependency "@helium/http@^3.43.0".
warning " > @helium/react-native-sdk@2.0.14" has incorrect peer dependency "@helium/transactions@^3.38.0".
warning " > @helium/react-native-sdk@2.0.14" has unmet peer dependency "events@^3.3.0".
warning " > @helium/react-native-sdk@2.0.14" has incorrect peer dependency "h3-js@^3.7.2".
warning " > @helium/react-native-sdk@2.0.14" has unmet peer dependency "react-native-crypto@^2.2.0".
warning " > @helium/react-native-sdk@2.0.14" has incorrect peer dependency "react-native-sodium@^0.3.9".
warning " > @helium/react-native-sdk@2.0.14" has unmet peer dependency "readable-stream@^3.6.0".
warning "@helium/react-native-sdk > react-native-crypto@2.2.0" has unmet peer dependency "react-native-randombytes@>=2.0.0 <4.0.0".
warning " > @helium/wallet-link@4.10.0" has unmet peer dependency "@helium/crypto@^4.3.1".
warning " > @react-native-mapbox-gl/maps@8.5.0" has unmet peer dependency "prop-types@>=15.5.8".
warning " > @react-navigation/stack@5.14.9" has unmet peer dependency "@react-native-community/masked-view@>= 0.1.0".
warning " > redux-thunk@2.4.2" has unmet peer dependency "redux@^4".
warning "expo-splash-screen > @expo/prebuild-config@5.0.7" has unmet peer dependency "expo-modules-autolinking@>=0.8.1".
warning "expo-camera > @koale/useworker@4.0.2" has incorrect peer dependency "react@^16.8.0".
warning "react-native > react-native-codegen > jscodeshift@0.13.1" has unmet peer dependency "@babel/preset-env@^7.1.6".
warning " > react-native-appstate-hook@1.0.6" has incorrect peer dependency "react@^16.8.0".
warning " > react-native-appstate-hook@1.0.6" has incorrect peer dependency "react-native@^0.59.0".
warning " > use-state-with-callback@2.0.5" has unmet peer dependency "react-dom@>=16.8.0".
warning " > eslint-config-react-app@5.2.1" has incorrect peer dependency "@typescript-eslint/eslint-plugin@2.x".
warning " > eslint-config-react-app@5.2.1" has incorrect peer dependency "@typescript-eslint/parser@2.x".
warning " > eslint-config-react-app@5.2.1" has unmet peer dependency "babel-eslint@10.x".
warning " > eslint-config-react-app@5.2.1" has incorrect peer dependency "eslint@6.x".
warning " > eslint-config-react-app@5.2.1" has unmet peer dependency "eslint-plugin-flowtype@3.x || 4.x".
warning " > eslint-config-react-app@5.2.1" has incorrect peer dependency "eslint-plugin-react-hooks@1.x || 2.x".
[4/4] Building fresh packages...
[-/9] ⠐ waiting...
[2/9] ⠐ bigint-buffer
[-/9] ⠐ waiting...
[-/9] ⠐ waiting...
error D:\MyProject\ugApplication\maker-starter-app-main\node_modules\@helium\react-native-sdk: Command failed.

**Comment by jackz123** [comment]
Error reported by the latest code (as shown below)
![image](https://user-images.githubusercontent.com/16610288/226339075-66a144dc-0c3f-4816-a3fe-a0516b33ecd7.png)

**Comment by matthewcarlreetz** [comment]
Can you update @helium/react-native-sdk to the latest?
`"@helium/react-native-sdk": "^2.0.18"`

**Comment by jackz123** [comment]
Yes, I have updated to  "^2.0.18"  and executed : rm -fr node_modules && yarn cache clean && yarn install
But the problem still exists.

![image](https://user-images.githubusercontent.com/16610288/227245039-c3373899-b618-4457-b31a-5cabfe3939e6.png)

**Comment by jackz123** [comment]
I also clean the project and delete the app, but the problem still exists
I pulled the latest code

![image](https://user-images.githubusercontent.com/16610288/227249173-d2097d1a-5150-4c48-9837-f90eacbe9359.png)

**Comment by jackz123** [comment]
There is no reference to react-native-quick-crypto in the code, but I re-imported react-native-quick-crypto, and configured babel, and it can run now

![image](https://user-images.githubusercontent.com/16610288/227257780-df10509d-907f-449d-87c9-b7dea4666b8b.png)
